outputs.md

```diff
@@ -17,64 +17,101 @@ FHD outputs various data products. We outline and describe these below. <br />
-
-**cal.auto_params**:
-
-**cal.convergence**:
-
-**cal.amp_params/cal.phase_params/cal.mode_params**: a pointer array of dimension N<sub>pol</sub> by N<sub>tile</sub> which points to an array of coefficients used in the polynomial or mode fitting. The amplitude and phase parameters are ordered by the least-degree to the highest-degree; a polynomial fit of A + Bx + Cx<sup>2</sup> has an array of three values A,B, and C. The mode parameters are ordered by mode in index units of the Fourier transform (can be a non-integer for the hyperfine DFT option), amplitude of the mode, and phase of the mode. The wave can be reconstructed by amp * e<sup>-i2pi*(mode*[0,1,2...n_freq-1]/n_freq)+i*phase</sup>. If mode parameters (or amplitude or phase parameters) are not calculated for that tile, there will be a Null pointer in that place.
-
-**cal.skymodel**: a structure of the various skymodel parameters used in calibration.
-  * **include_calibration**: !Q, **n_sources**: number of sources included in the calibration skymodel, **catalog_name**: name of the catalog used in the calibration, **galaxy_model**: integer 1/0 value of whether or not a galaxy model was used, **galaxy_spectral_index**: the spectral index of the galaxy model used or NaN if no galaxy model was used, **diffuse_model**: name of the diffuse model used in the calibration, **diffuse_spectral_index**: the spectral index of the diffuse model used of NaN if no diffuse model was used. Source_list defined below.
-
-**cal.skymodel.source_list**: an array of structures of dimension N<sub>sources</sub>.
-  * **x**: !Q, **y**: !Q, **ra**: the RA coordinate value of the source, **dec**: the DEC coordinate value of the source, **ston**: not used in the calibration source list (!Q right?), **freq**: frequency of the source model, **alpha**: spectral index of the source, **gain**: not used in the calibration source list (!Q right?), **flag**: type codes where 0 is no flag, 1 is low confidence, and 2 is sidelobe contamination
-  * **extend**: !Q 
-  * **flux**: structure of the fluxes for that source, where the dimension depends on what polarizations are reported for that source. The order of potential polarizations is xx, yy, xy, and yx in apparent brightness or I, Q, U, V in sky brightness.
+**cal**: a structure of the various calibration parameters. Pointers and large arrays in this structure are defined below. Other variables in the structure include:
+  * **n_pol**: long number of polarizations
+  * **n_freq**: long number of frequency channels
+  * **n_tile**: long number of tiles
+  * **n_time**: long number of time steps
+  * **auto_initialize**: integer 1/0 value of whether window scaled autocorrelations were used/not used
+  * **max_iter**: long number of maximum iterations in the X<sup>2</sup> calibration subroutine
+  * **phase_iter**: long number of maximum iterations in the X<sup>2</sup> calibration subroutine where only the phase was updated
+  * **auto_scale**: !Q
```

Comment:
The average amplitude of the calibration solutions fit from the auto-correlations.